OpenEdX/Managers/FirebaseAnalyticsService/FirebaseAnalyticsService.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  FirebaseAnalyticsService.swift
+//  OpenEdX
+//
+//  Created by Anton Yarmolenka on 19/02/2024.
+//
+
+import Foundation
+import Firebase
+import Core
+
+class FirebaseAnalyticsService: AnalyticsService {
+    // Init manager
+    public init(config: ConfigProtocol) {
+        guard config.firebase.enabled && config.firebase.isAnalyticsSourceFirebase else { return }
+
+        FirebaseApp.configure()
```

Comment:
Since we don't have the GoogleServices.plist file to configure Firebase, we have to pass the configuration to the configure function.
https://github.com/openedx/openedx-app-ios/blob/main/OpenEdX/AppDelegate.swift#L42

Reply:
@volodymyr-chekyrta  The [config processing script](https://github.com/openedx/openedx-app-ios/blob/main/config_script/process_config.py#L156) is already generating the `GoogleService-Info.plist` file by reading the values from the config for `FIREBASE`.

Reply:
I see, thank you!